oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java

```diff
@@ -216,6 +216,8 @@ static FullGCMode getFullGcMode() {
     private final boolean fullGCEnabled;
     private final boolean isFullGCDryRun;
     private final boolean embeddedVerification;
+    private Set<String> fullGCIncludePaths;
```

Comment:
Maybe make these "@NonNull"?

Reply:
done in https://github.com/apache/jackrabbit-oak/pull/1547/commits/14ba933fad9b8dadcd51d00aa2e186a0b3e39568